website/layouts/robots.txt

```diff
@@ -0,0 +1,8 @@
+User-agent: *
+
+Disallow: /v0.4.3-docs/
```

Comment:
This is going to get stale. We need a longer term solution to avoid updating this on every release. Perhaps we can simply allow `/docs`.